website/content/en/docs/config/environment-variables.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Environment Variables
+weight: 6
+---
+
+## Environment Variables
+
+This page documents the environment variables used in Lima.
+
+### `LIMA_INSTANCE`
+
+- **Description**: Specifies the name of the Lima instance to use.
+- **Default**: `default`
+- **Usage**: 
+  ```sh
+  export LIMA_INSTANCE=my-instance
+  lima uname -a
+  ```
+- **Used in**: `cmd/lima`, `cmd/lima.bat`, `cmd/docker.lima`, `cmd/kubectl.lima`
+
+### `LIMA_SHELL`
+
+- **Description**: Specifies the shell interpreter to use inside the Lima instance.
+- **Default**: User's shell configured inside the instance
+- **Usage**: 
+  ```sh
+  export LIMA_SHELL=/bin/bash
+  lima
+  ```
+- **Used in**: `cmd/lima`, `cmd/limactl/shell.go`
```

Comment:
I don't think this file is used in `cmd/limactl/shell.go`.
Even if it was used, such an implementation detail would not need to be documented here.

Reply:
The same applies to other `*.go` source files

Reply:
![image](https://github.com/user-attachments/assets/15f6fdbd-8d22-4e0a-9525-c86ab7766c7a)

`.go` files are mentioned here only. This PR is created mainly for this environment var only.

Reply:
The implementation details shouldn't be mentioned in user docs

Reply:
Which line are you referring to?

Reply:
usernet.go